include/fmt/format.h

```diff
@@ -1593,7 +1593,7 @@ template <typename OutputIt, typename Char, typename UInt> struct int_writer {
     format_decimal(digits, abs_value, num_digits);
     basic_memory_buffer<Char> buffer;
     size += prefix_size;
-    buffer.resize(size);
+    buffer.resize(static_cast<std::size_t>(size));
```

Comment:
I suggest using to_unsigned instead of a cast.

Reply:
Yupp, I agree.
Is done in a2c668fcaafd92afddb3837bdd36573006c4a69c.